src/app/components/breadcrumb/breadcrumb.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { BreadcrumbComponent } from './breadcrumb.component';
+import { BreadcrumbService } from './breadcrumb.service';
+
+@NgModule({
+  imports: [
+    CommonModule
+  ],
+  declarations: [BreadcrumbComponent]
```

Comment:
You are missing the export for `BreadcrumbComponent`. Without this export the module won't work on its own. In your example `app.module.ts` you declare the component and service manually,